vscode/package.json

```
@@ -546,7 +554,12 @@
       {
         "command": "cody.chat.panel.new",
         "key": "alt+l",
-        "when": "cody.activated"
+        "when": "cody.activated && !editorHasSelection"
+      },
+      {
+        "command": "cody.chat.context.add",
+        "key": "alt+l",
+        "when": "cody.activated && !editorReadonly && editorTextFocus && editorHasSelection"
```

```suggestion
      {
        "command": "cody.chat.context.add",
        "key": "alt+l",
        "when": "cody.activated && editorTextFocus && editorHasSelection"
```

Shouldn't matter if this is `editorReadonly` for chat, right?

---

it does because we will need to read the text from the editor but not all the readOnly editor are local files. For example, using a remote file created by the Sourcegraph extension, the command doesn't work if we remove `!editorReadonly &&`:

https://github.com/sourcegraph/cody/assets/68532117/a613c04a-c256-495d-b3db-0ec39b14512d